modules/my_health/spec/requests/my_health/v1/messaging/sm_spec_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rspec/rails'
+
+RSpec.configure do |config|
+  config.before(:suite) do
+    # Ensure that Flipper is disabled globally for all tests
+    Flipper.disable(:mhv_secure_messaging_cerner_pilot)
+  end
+
+  config.after do
+    # Ensure that Flipper is disabled globally for all tests
+    Flipper.disable(:mhv_secure_messaging_cerner_pilot)
```

Comment:
allow(Flipper).to receive(:enabled?).with(: mhv_secure_messaging_cerner_pilot, instance_of(User)).and_return(false)